e3nn_jax/_src/s2grid_test.py

```diff
@@ -8,7 +8,8 @@
 
 
 @pytest.mark.parametrize("quadrature", ["soft", "gausslegendre"])
-def test_s2grid_transforms(quadrature):
+@pytest.mark.parametrize("naive", [False, True])
+def test_s2grid_transforms(quadrature, naive):
```

Comment:
Could you also test naive vs fft?

```suggestion
@pytest.mark.parametrize("naive_to", [False, True])
@pytest.mark.parametrize("naive_from", [False, True])
def test_s2grid_transforms(quadrature, naive_to, naive_from):
```

Or create a new test that compare naive vs fft? As we wish